specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/managedClusters.json

```diff
@@ -6795,6 +6799,16 @@
       },
       "description": "The security settings of an agent pool."
     },
+    "AgentPoolGPUPProfile": {
+      "type": "object",
+      "properties": {
+        "installGPUDrivers": {
```

Comment:
this should match the variable name we defined in 2023-09-02-preview AKS RP API. It should be 'installGPUDriver'